### ISSUE TITLE
Nengo_gui stops updating graphs when running with nengo_spinnaker backend

### ISSUE THREAD

**Issue by pabogdan** [nengo/nengo_gui#608]
After 10 seconds, the simulation on SpiNNaker pauses to retrieve data, but after that it stops updating semantic pointer clouds and some value graphs for ensembles.
```bash
INFO:nengo_spinnaker.simulator:Retrieving simulation data
INFO:nengo_spinnaker.simulator:Retrieving data took 0.000963 seconds
INFO:nengo_spinnaker.simulator:Running simulation...
```
Not completely sure who needs to find a fix, so @mundya I hope you could confirm if it's a nengo_spinnaker issue or otherwise.

**Comment by pabogdan** [comment]
bump, this is still happening

**Comment by mundya** [comment]
This should have been fixed by project-rig/nengo_spinnaker#88, at least that fixed it for me. Can you check and I'll have another look tomorrow?

**Comment by pabogdan** [comment]
@mundya You are right, it is working now. It was a problem of not running setup.py again after pulling to more recent versions.
